modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/loginstance.yaml

```diff
@@ -8,20 +8,17 @@ metadata:
 spec:
   clients:
     - url: {{ .Values.remote.logsUrl }}
+      tenantId: {{ .Values.tenantId }}
       externalLabels:
         cluster: {{ .Values.clusterName }}
         environment: {{ .Values.environment }}
-      basicAuth:
-        username:
-          name: {{ .Values.credentialsSecretName }}
-          key: logs_username
-        password:
-          name: {{ .Values.credentialsSecretName }}
-          key: logs_password
+      tlsConfig:
+        certFile: /mnt/secret-store/tls.crt
+        keyFile: /mnt/secret-store/tls.key
   podLogsNamespaceSelector:
     matchExpressions:
       - key: "xkf.xenit.io/kind"
         operator: "Exists"
```

Comment:
Don't think you will be able to use `podLogsNamespaceSelector` thanks to all the interesting namespace don't use our labels. Especially thinking of kube-system.

```shell
$ k get ns -L xkf.xenit.io/kind
NAME                               STATUS   AGE    KIND
aad-pod-identity                   Active   160d   platform
azad-kube-proxy                    Active   142d   platform
azure-metrics                      Active   71d    platform
calico-system                      Active   160d   
cert-manager                       Active   160d   platform
csi-secrets-store-provider-azure   Active   160d   platform
default                            Active   160d   
external-dns                       Active   160d   platform
falco                              Active   160d   platform
flux-system                        Active   157d   platform
gatekeeper-system                  Active   160d   platform
grafana-agent                      Active   71d    platform
ingress-healthz                    Active   160d   platform
ingress-nginx                      Active   160d   platform
kube-node-lease                    Active   160d   
kube-public                        Active   160d   
kube-system                        Active   160d   
prometheus                         Active   160d   platform
reloader                           Active   160d   platform
service-accounts                   Active   160d   
starboard-operator                 Active   71d    platform
tigera-operator                    Active   160d   
vpa                                Active   56d    platform
```

Reply:
Correct, I'll remove